<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>822</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>502</width>
    <height>822</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Biblioteca</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Amministratore/Resources/logo.png</normaloff>../../Amministratore/Resources/logo.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>800</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>800</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginPage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>700</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>700</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(View/Amministratore/Resources/background.png);
border-radius: 50px;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 100);
border-image: None;</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item alignment="Qt::AlignTop">
              <widget class="QWidget" name="header" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: None;</string>
               </property>
               <widget class="QLabel" name="head">
                <property name="geometry">
                 <rect>
                  <x>130</x>
                  <y>20</y>
                  <width>162</width>
                  <height>60</height>
                 </rect>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>162</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>30</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Log In</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="body" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: None;</string>
               </property>
               <widget class="QLineEdit" name="matricolaInput">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>80</y>
                  <width>355</width>
                  <height>40</height>
                 </rect>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0,0,0,0);
border: None;
border-bottom: 2px solid rgb(255, 255, 255);
color: rgba(255, 255, 255, 230);
padding-bottom: 7px;</string>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhNone</set>
                </property>
                <property name="placeholderText">
                 <string>Matricola</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="passwordInput">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>180</y>
                  <width>355</width>
                  <height>40</height>
                 </rect>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0,0,0,0);
border: None;
border-bottom: 2px solid rgb(255, 255, 255);
color: rgba(255, 255, 255, 230);
padding-bottom: 7px;</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Password</string>
                </property>
               </widget>
               <widget class="QPushButton" name="loginButton">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>295</y>
                  <width>355</width>
                  <height>50</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton#loginButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius: 15px;
}
QPushButton#loginButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#loginButton:pressed {
	background-color: rgba(105,118,132,200);
	padding-left: 5px;
	padding-top: 5px;
}</string>
                </property>
                <property name="text">
                 <string>Sign In</string>
                </property>
                <property name="shortcut">
                 <string/>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
               <widget class="QLabel" name="infoLabel">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>410</y>
                  <width>351</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Sei un nuovo studente? </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QPushButton" name="registrazioneStudenteButton">
                <property name="geometry">
                 <rect>
                  <x>280</x>
                  <y>410</y>
                  <width>91</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton#registrazioneStudenteButton{
	color: rgb(0, 170, 255);
}
QPushButton#registrazioneStudenteButton:pressed{	
	color: rgb(0, 86, 129);
}
</string>
                </property>
                <property name="text">
                 <string>Registrati</string>
                </property>
               </widget>
               <widget class="QLabel" name="errorLabel">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>20</y>
                  <width>355</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <italic>true</italic>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(185, 0, 0);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="infoLabel_2">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>440</y>
                  <width>351</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Sei un nuovo amministratore? </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QPushButton" name="registrazioneAmministratoreButton">
                <property name="geometry">
                 <rect>
                  <x>300</x>
                  <y>440</y>
                  <width>93</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton#registrazioneAmministratoreButton{
	color: rgb(0, 170, 255);
}
QPushButton#registrazioneAmministratoreButton:pressed{	
	color: rgb(0, 86, 129);
}
</string>
                </property>
                <property name="text">
                 <string>Registrati</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/setup.qrc"/>
 </resources>
 <connections/>
</ui>
